bentoml/yatai/web/src/server.ts

```diff
@@ -39,6 +39,22 @@ const createRoutes = (app, yataiClient) => {
       return res.status(500).json(error);
     }
   });
+  
+  router.post("/api/metrics", async (req: Request, res:Response) => {
```

Comment:
It is a common convention to use `/metrics` rather than `/api/metrics`. See BentoService's API server's implementation. https://github.com/bentoml/BentoML/blob/1d885119ae8e560b4fbd60f0de80f08383ab0a05/bentoml/server/api_server.py#L269